app/src/main/java/ingsw/group1/findmyphone/fragment/HomeFragment.java

```diff
@@ -0,0 +1,110 @@
+package ingsw.group1.findmyphone.fragment;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.EditText;
+import android.widget.ImageButton;
+import android.widget.Toast;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.navigation.Navigation;
+
+import com.eis.smslibrary.SMSPeer;
+
+import ingsw.group1.findmyphone.Manager;
+import ingsw.group1.findmyphone.R;
+
+/**
+ * @author Turcato, Kumar
+ */
+public class HomeFragment extends Fragment {
+
+    private static final String INVALID_ADDRESS_MESSAGE = "The provided phone address is invalid";
+
+    private EditText addressInput;
+    private EditText passwordInput;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'passwordInput' could be replaced by a local variable.](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=5082269)